test/index.js

```diff
@@ -1,20 +1,145 @@
 'use strict'
 
-var test = require('tape')
-var fs = require('fs')
-var path = require('path')
-var rimraf = require('rimraf')
+const test = require('tape')
+const fs = require('fs')
+const path = require('path')
+const rimraf = require('rimraf')
+const goenv = require('go-platform')
+const pkg = require('./../package.json')
+const Download = require('../src')
```

Comment:
Nope, not a class, just a function. See couple of lines below in the test file why that is wrapped like that.